File: src/main/kotlin/com/gyh/fleacampus/mapper/UserMapper.java
package com.gyh.fleacampus.mapper;


import com.gyh.fleacampus.model.User;

import java.util.List;

public interface UserMapper {
    User loadUserByUsername(String username);

    User findUserRoleById(Integer id);

    User findUserById(Integer id);

    List<User> findAll();

    int save(User user);

    Integer deleteUserById(Integer id);

    Integer update(User user);
}

File: src/main/kotlin/com/gyh/fleacampus/mapper/UserMapper.kt
package com.gyh.fleacampus.mapper

import com.gyh.fleacampus.model.User

interface UserMapper {
    fun deleteByPrimaryKey(id: Int): Int
    fun insert(record: User): Int
    fun insertSelective(record: User): Int
    fun selectByPrimaryKey(id: Int): User?
    fun updateByPrimaryKeySelective(record: User): Int
    fun updateByPrimaryKey(record: User): Int
    fun loadUserByUsername(username: String): User?
}